ui/src/App.js

```diff
@@ -1,15 +1,32 @@
-import Dashboard from "./Components/Dashboard/Dashboard";
-import { FEATURE_FLAGS, FEATURE_FLAGS_HEADERS} from "./Mock/featureFlags";
+import { useState } from 'react';
+import Dashboard from './Components/Dashboard/Dashboard';
+import Filters from './Components/Filters/Filters';
+import Search from './Components/Search/Search';
+import { useFilterContext } from './Context/filter_context';
+import { FEATURE_FLAGS, FEATURE_FLAGS_HEADERS } from './Mock/featureFlags';
 
 function App() {
+  const { filteredList, isLoading, featureFlagHeader, clearFilter } =
+    useFilterContext();
+
+  if (isLoading) {
+    // We can add spinner here - TODO
+    return <div>Loading...</div>;
+  }
+
   return (
-    <main 
-      className='flex flex-col items-center'
-    >
-      <Dashboard 
-        featureFlags={FEATURE_FLAGS} 
-        headers={FEATURE_FLAGS_HEADERS}
-      />
+    <main className='flex flex-col items-center'>
+      <section className='flex justify-center items-center '>
+        <Search />
+        <Filters />
+      </section>
+      <button
```

Comment:
`button` should be wrapped within the section of  `Filters` and `Search`

Reply:
button moved inside section